frontend/src/Components/ReleasePlan/Sprint.jsx

```diff
@@ -50,6 +50,17 @@ const Sprint = ({ index, items, setItems, userStories }) => {
 			.then((response) => response.json())
 			.then((result) => {
 				setStories(result)
+				console.log(items)
+				// console.log(result)
```

Comment:
could you delete these logs? There are already a lot of things being spammed in the console so in the future it would be hard to tell which ones we need